network/discovery/dv5_service.go

```diff
@@ -331,15 +354,18 @@ func (dvs *DiscV5Service) createLocalNode(logger *zap.Logger, discOpts *Options,
 		return nil, errors.Wrap(err, "could not add configured addresses")
 	}
 	f := forksfactory.NewFork(discOpts.ForkVersion)
-	err = f.DecorateNode(localNode, map[string]interface{}{
-		"operatorID": opts.OperatorID,
-		"subnets":    opts.Subnets,
-	})
+	err = f.DecorateNode(
+		localNode,
+
+		// Satisfy decorations of forks supported by this node.
+		genesis.DecorateWithDomainType(dvs.domainType),
+		genesis.DecorateWithSubnets(opts.Subnets),
```

Comment:
should genesis be hardcoded? shouldn't it be derived from the domain type? probably relates to what @olegshmuelov is working on right now.

Reply:
now that forks are removed, these became package methods